Makefile

```diff
@@ -113,6 +119,21 @@ create-kubeconfig-cm:
 delete-kubeconfig-cm:
 	kubectl -n kube-system delete configmap $(KUBECONFIG_CM)
 
+.PHONY : tls-secret
```

Comment:
Is it "tls-secrets"  instead of "tls-secret" ?

Reply:
It's a typo. Should be `tls-secrets`